src/test/java/edu/matc/covidPulse/controller/AllServiceTest.java

```diff
@@ -0,0 +1,54 @@
+package edu.matc.covidPulse.controller;
```

Comment:
If we're throwing out the AllService controller, we should throw out the corresponding test class as well. If these tests are still useful, and not something already covered in Matt's tests, we can move them over to his test class.

Reply:
I'll delete the files in my branch.